### ISSUE TITLE
typo under cuda?

### ISSUE THREAD

**Issue by scothalverson** [StanfordLegion/legion#518]
When compiling Flecsi under Clang CUDA with Legion as the backend, I'm getting the following error:
```
error: use of undeclared identifier 'has_transfrom'; did you mean 'has_transform'?
```
One instance that results in this error can be found below.

https://github.com/StanfordLegion/legion/blob/62b24fd6cb012d863ac68618407db9b3cac3374b/runtime/legion/legion.inl#L1427

There are 3 instances of `has_transfrom` in that file, at lines 1427, 1711, and 2005.  

Replacing these instances with `has_transform` allows me to continue compiling, so I assume the error message is correct.

**Comment by lightsighter** [comment]
Yes, that's a typo and should be fixed in fefaf45 on gitlab in the master branch. Will be over here on github in the master branch in a little bit. Let me know if you find other typos. Not all of these templates have been instantiated before.

**Comment by scothalverson** [comment]
After these two issues, I was able to get Flecsi to compile and link, so I think we're good for now.  Thanks Mike.
